Mathlib/LinearAlgebra/Dimension/Finrank.lean

```diff
@@ -50,8 +50,8 @@ For a vector space `M` over a field `R`, this is the same as the finite dimensio
 of `M` over `R`.
 
 Note that it is possible to have `M` with `¬(Module.Finite R M)` but `finrank R M ≠ 0`, for example
-`ℤ × ℚ/ℤ` has `finrank` equal to `1`.
--/
+`ℤ × ℚ/ℤ` has `finrank` equal to `1`. -/
+@[stacks 09G3 "second part"]
```

Comment:
Is this the second part? The second part seems to be defining what finite extensions are.

Reply:
Thanks! Removed.